src/components/MultiDropdown/MultiDropdown.module.scss

```
@@ -0,0 +1,92 @@
+.multiDropdown {
+  display: inline-block;
+  position: relative;
+  box-sizing: border-box;
+  font-family: 'Circular Std', sans-serif;
+  font-size: 12px;
+  max-width: 320px;
+  min-height: 22px;
+
+  &__button {
+    position: relative;
+    font-size: 12px;
+    display: block;
+    color: #343A40;
+    width: 100%;
+    min-width: 226px;
+    text-align: left;
+    background-color: #FFFFFF;
+    border: 0.5px solid #DFE2E4;
+    padding: 5px 8px;
+    padding-right: 20px;
+    border-radius: 12px;
+    background-image: url('./images/Vector.svg');
+    background-repeat: no-repeat;
+    background-position: 97.5% 50%;
+    background-size: 7px;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    cursor: pointer;
+
+    &:hover {
+      border: 0.5px solid #6C757D;
+    }
+
+    &.dropdown_disabled {
+      background-color: #F8F9FA;
+      border: 0.5px solid #DFE2E4;
+    }
+  }
+
+  &__list {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    top: 30px;
+    left: 0;
+    border-radius: 12px;
+    width: 100%;
+    max-height: 125px;
+    overflow: auto;
+    cursor: pointer;
+    border: 0.5px solid #DFE2E4;
+    z-index: 100;
+
+    &_label {
+      position: relative;
+      border-bottom: 0.5px solid #DFE2E4;
+
+      &:first-child {
+        border-radius: 12px 12px 0 0;
+      }
+
+      &:last-child {
+        border-radius: 0 0 12px 12px;
+        border-bottom: 0px;
+      }
+
+    }
+
+    &_input {
+      position: absolute;
+      appearance: none;
+
+      &:checked+.multiDropdown__list_item {
+        background-color: #F8F9FA;
+      }
+    }
+
```

мне кажется, селектор `&:checked+.multiDropdown__list_item` некорректен, так как `.multiDropdown__list` и `&__list` в данном случае, скорее всего, разные классы. кажется, вот это будет работать так, как предполагается:
```suggestion
    }
    
    &_input:checked#{&}_item {
      background-color: #F8F9FA;
    }
```

то есть это преобразуется в `.multiDropdown__list_input:checked.multiDropdown__list_item { ... }` с учётом всех модификаций названий классов, которые применяют модули

обёртка в #{ } чтобы scss не ругался и просто